script/redirect-cc.py

```diff
@@ -22,6 +23,13 @@ def main():
     return cc_retcode
 
 
+def append_root_src_include(args):
+  for arg in args:
+    if arg.startswith('-I') and arg.endswith('brave/chromium_src'):
+        args.append(arg + '/../../..')
```

Comment:
Would it be possible to remove one level from `/../../..` (i.e. pass `/../..` only) here so that we don't need to prepend every include in https://github.com/brave/brave-core/pull/11126/commits/61cfda425c794f590479a05d97e387ba227be798 with a `src/` ?

Reply:
nope, in this case it will be impossible to distinguish between `brave/chromium_src` and original files.

Reply:
Ah,I see thanks for the clarification. In any case, this was confusing to me reading because I was expecting /../.. instead, so maybe you could add a comment here clarifying why this is done that way?

Reply:
thanks, I guess the function name was a little off. Updated it and added a comment.